src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,21 @@
-export const CommentInfo = () => <>Put the comment here</>;
+export const CommentInfo = ({ comment }) => {
+  const { name, email, body } = comment;
+
+  return (
+    <>
```

Comment:
Why do you wrap this component into the fragment? (here and in other components)